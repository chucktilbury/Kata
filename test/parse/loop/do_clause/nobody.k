;-------------------------------------------------------------------------------
; do/while with no body rejected
;-------------------------------------------------------------------------------

func name ()() {

    do while
}
