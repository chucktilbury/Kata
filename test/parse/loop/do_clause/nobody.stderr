Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/nobody.k: 7: 13: expected a function body but got while
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/nobody.k: 9: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/nobody.k: 9: 1: expected function body but got end of file
